tests/python/pytorch/graphbolt/impl/test_negative_sampler.py

```diff
@@ -5,6 +5,75 @@
 from torchdata.datapipes.iter import Mapper
 
 
+def test_NegativeSampler_invoke():
+    # Instantiate graph and required datapipes.
+    num_seeds = 30
+    item_set = gb.ItemSet(
+        torch.arange(0, 2 * num_seeds).reshape(-1, 2), names="node_pairs"
+    )
+    batch_size = 10
+    item_sampler = gb.ItemSampler(item_set, batch_size=batch_size)
+    negative_ratio = 2
+
+    # Invoke NegativeSampler via class constructor.
+    negative_sampler = gb.NegativeSampler(
+        item_sampler,
+        negative_ratio,
+        gb.LinkPredictionEdgeFormat.INDEPENDENT,
+    )
+    with pytest.raises(NotImplementedError):
+        next(iter(negative_sampler))
+
+    # Invoke NegativeSampler via functional form.
+    negative_sampler = item_sampler.sample_negative(
+        negative_ratio,
+        gb.LinkPredictionEdgeFormat.INDEPENDENT,
+    )
+    with pytest.raises(NotImplementedError):
+        next(iter(negative_sampler))
+
+
+def test_UniformNegativeSampler_invoke():
+    # Instantiate graph and required datapipes.
+    graph = gb_test_utils.rand_csc_graph(100, 0.05)
+    num_seeds = 30
+    item_set = gb.ItemSet(
+        torch.arange(0, 2 * num_seeds).reshape(-1, 2), names="node_pairs"
+    )
+    batch_size = 10
+    item_sampler = gb.ItemSampler(item_set, batch_size=batch_size)
+    negative_ratio = 2
+
+    # Verify iteration over UniformNegativeSampler.
+    def _verify(negative_sampler):
+        for data in negative_sampler:
+            src, dst = data.node_pairs
+            labels = data.labels
+            # Assertation
+            assert len(src) == batch_size * (negative_ratio + 1)
+            assert len(dst) == batch_size * (negative_ratio + 1)
+            assert len(labels) == batch_size * (negative_ratio + 1)
+            assert torch.all(torch.eq(labels[:batch_size], 1))
+            assert torch.all(torch.eq(labels[batch_size:], 0))
+
+    # Invoke UniformNegativeSampler via class constructor.
+    negative_sampler = gb.UniformNegativeSampler(
+        item_sampler,
+        negative_ratio,
+        gb.LinkPredictionEdgeFormat.INDEPENDENT,
```

Comment:
Conflict with https://github.com/dmlc/dgl/pull/6298, please be careful when merging.

Reply:
Feel free to merge, and ask @peizhou001 to clean this up in #6298